app/components/project-recommendation-new.js

```diff
@@ -0,0 +1,126 @@
+    dateReceived: new Date('2015-12-12T03:24:00'),
+    dateVoted: new Date('2014-12-17T01:23:45'),
+    isActive: 'active',
+    status: 'Submitted',
+    docketDescription: 'My awesome docket description.',
+    // Community-Board specific
+    votesInFavor: 77,
+    votesAgainst: 22,
+    votesAbstain: 25,
+    totalBoardMembers: 14,
+    didQuorumExist: false,
+  });
+
+  newRec.save();
+}
+
+function createBoroughBoardRecForActions(thisRec) {
+  const newRec = thisRec.store.createRecord('borough-board-recommendation', {
+    // general
+    user: thisRec.user,
+    // assume all actions for now
+    // TODO: filter actions by user's selection
+    actions: thisRec.project.get('actions'),
+    recommendation: 'Approved with Modifications/Conditions',
+    consideration: 'My awesome memo.',
+    voteLocation: '123 Boston Ave, Brooklyn, NY 11111',
+    dateReceived: new Date('2015-21-12T03:24:00'),
+    dateVoted: new Date('2014-12-17T01:23:45'),
+    isActive: 'active',
+    status: 'Submitted',
+    docketDescription: 'My awesome docket description.',
+    // Community-Board specific
+    votesInFavor: 77,
+    votesAgainst: 22,
+    votesAbstain: 25,
+    totalBoardMembers: 14,
+    didQuorumExist: false,
+  });
+
+  newRec.save();
+}
+
+// Creates a new Recommendation based on a combination of
+// user inputted Recommendation attributes
+// and associates it to a given User, Project, and set of selected Actions
+// TODO: Hook up this component to user inputted data.
+export default class ProjectRecommendationNewComponent extends Component {
+  @service
+  store;
+
+  // Logged in User
+  // TODO: Get this from the auth service.
+  // Also, retrieve by email?
+  user = this.store.findRecord('user', 2, {
+    reload: true,
+    include: 'userProjectParticipantTypes.project,projects',
+  });
+
+  // A ZAP Project
+  // TODO: Acquire outside this component.
+  project = this.store.findRecord('project', 3, {
+    reload: true,
+  });
+
+  // Should the Recommendation type depend on
+  // User.landUseParticipant, User.participant,
+  // or User.userProjectPartipantTypes[0].participantType?
+  @action
+  submitRec() {
+    const thisRec = this;
+    // TODO: Waiting for this promises to resolve should
+    // occur outside this component
+    Promise.all([thisRec.user, thisRec.project]).then(function (values) {
+      const [recUser, recProject] = values;
+      // Q: What happens when a given User and Project has two different UserProjectParticipantType.participantType values?
+      // (assuming the Recommendation to create depends on UserProjectParticipantType.participantType)
+      const userProjectParticipantType = recUser.get('userProjectParticipantTypes').filterBy('project.id', recProject.get('id')).firstObject;
+      const userParticipantType = userProjectParticipantType.get('participantType');
+      if (userParticipantType === 'BP') {
+        createBoroughPresidentRecForActions(thisRec);
```

Comment:
@allthesignals The frontend logic here will create a different type of Recommendation based on User.UserProjectParticipantType.participantType

Reply:
thanks!